Makefile.docs

```diff
@@ -36,7 +37,7 @@ docs-api: install-ci
 	done;
 
 docs-site:
-	$(MAKE) -C $(SITE_GENERATOR) docs output=$(DESTINATION_FOLDER) shared=$(SHARED_FOLDER) baseurl=$(BASE_URL) owner=$(REPOSITORY_OWNER) repository=$(REPOSITORY) version=$(VERSION_NAME) languages=$(LANGUAGES)
+	$(MAKE) -C $(SITE_GENERATOR) docs OUTPUT=$(SITE_DOCS_FOLDER) SHARED=$(SHARED_FOLDER) BASEURL=$(BASE_URL) OWNER=$(REPOSITORY_OWNER) REPOSITORY=$(REPOSITORY) VERSION=$(VERSION_NAME) LANGUAGES="$(LANGUAGES)" SOURCES=$(PWD)
```

Comment:
have you tried this? pretty sure `$(PWD)` does not do what you expect (it's empty), you probably mean `$(shell pwd)`.